app/uk/gov/hmrc/nisp/views/statepension_forecastonly.scala.html

```diff
@@ -82,50 +69,67 @@ <h2>@messages("nisp.nirecord.helpline.getHelp")</h2>
-        @statePensionAgeUnderConsideration(langUtils.Dates.formatDate(statePension.pensionDate))
-    }
-
-    @if(statePension.contractedOut) {
-        @contractedOut()
-    }
-
-    @if(customerAge > Constants.deferralCutOffAge) {
-        @deferral(langUtils.Dates.formatDate(statePension.pensionDate))
-    }
-
-    @if(isPertaxUrl){<div class="panel-indent">@Html(messages("nisp.landing.estimateprovided"))</div>}
+            <div class="page-info">@user.name.getOrElse("")</div>
+            <h1 class="titleWithPageInfo">@messages("nisp.main.h1.title")</h1>
+
+            <p>@messages("nisp.national.insurance.number")<span class="display-block bold">@user.nino</span></p>
+
+            <div class="highlighted-event govuk-box-highlight">
+                <p>@messages("nisp.main.basedOn") <span class="nowrap">@langUtils.Dates.formatDate(statePension.pensionDate). </span> <span class="nowrap">@messages("nisp.main.caveats") @messages("nisp.is")</span></p>
+                <p><em>@NispMoney.pounds(statePension.amounts.forecast.weeklyAmount) @messages("nisp.main.week")</em></p>
+                <p>@NispMoney.pounds(statePension.amounts.forecast.monthlyAmount) @messages("nisp.main.month"),
+                @NispMoney.pounds(statePension.amounts.forecast.annualAmount) @messages("nisp.main.year")</p>
+            </div>
+            <p>@messages("nisp.main.caveats")</p>
+            <ul class="list-bullet">
+                <li>@Html(messages("nisp.main.notAGuarantee"))</li>
+                <li>@Html(messages("nisp.main.isBased", langUtils.Dates.formatDate(statePension.earningsIncludedUpTo)))</li>
+                <li>@Html(messages("nisp.main.inflation"))</li>
+                @if(statePension.pensionSharingOrder) {<li>@Html(messages("nisp.main.psod"))</li>}
+            </ul>
+            <h2 class="heading-medium" id="mostYouCanGet">@NispMoney.pounds(statePension.amounts.forecast.weeklyAmount) @Html(messages("nisp.main.mostYouCanGet"))</h2>
+            <p>@Html(messages("nisp.main.cantImprove"))</p>
+            <p>@Html(messages("nisp.main.context.reachMax.needToPay", (langUtils.Dates.formatDate(statePension.pensionDate)).toString()))</p>
+            <a href='@routes.NIRecordController.showFull.url'>@Html(messages("nisp.main.showyourrecord"))</a>
+
+            @if(isAbroad) {
+                @abroad()
+            }
 
+            @if(statePension.statePensionAgeUnderConsideration) {
+                @statePensionAgeUnderConsideration(langUtils.Dates.formatDate(statePension.pensionDate))
+            }
+
+            @if(statePension.contractedOut) {
+                @contractedOut()
+            }
+
+            @if(customerAge > Constants.deferralCutOffAge) {
+                @deferral(langUtils.Dates.formatDate(statePension.pensionDate))
+            }
+
+            @if(isPertaxUrl){<div class="panel-indent">@Html(messages("nisp.landing.estimateprovided"))</div>}
+        </div>
+
+        <div class="column-one-third">
```

Comment:
if this sidebar is the same for every page, it may be worth adding this to a partial/shared component and calling it, instead of repeating it in the different views. Might save time over editing multiple pages, just a thought.

Reply:
can do. only thing is that in 2 views its the same and in one view (statepension_cope.scala.html) its got a back link as well. your call if you want me to put it in template or leave it as it is?

Reply:
👍